packages/wdio-cli/src/commands/config.js

```diff
@@ -15,10 +15,27 @@ export const builder = {
     }
 }
 
-export const runConfig = async function (useYarn, exit) {
+export const runConfig = async function (useYarn, yes, exit) {
     console.log(CONFIG_HELPER_INTRO)
+    let answers
+    if (yes) {
+        QUESTIONNAIRE.forEach((question) => {
+            answers = answers || {}
+            if (question.when && !question.when(answers)) return
+
+            if (question.default) {
+                answers[question.name] = question.default
```

Comment:
I think overall we can simplify this:

```js
answers[question.name] = question.default
    /**
     * set default value if existing
     */
    ? question.default
    : question.choices && question.choices.length
        /**
         * pick first choice
         */
        ? question.choices[0].value
        : question.choices[0]
```

Reply:
@christian-bromann I adjusted this a little to make it work:
```
            answers[question.name] = question.default
            /**
             * set default value if existing
             */
                ? question.default
                : question.choices && question.choices.length
                /**
                 * pick first choice, select value if it exists
                 */
                    ? question.choices[0].value
                        ? question.choices[0].value
                        : question.choices[0]
                    : null
```

What do you think?